parsers/IN_WE_new.py

```diff
@@ -0,0 +1,257 @@
+import json
+from datetime import datetime, timedelta
+from logging import Logger, getLogger
+from typing import Optional
+
+import arrow
+import pandas as pd
+import pandasql as psql
+from requests import Response, Session
+
+from parsers.lib.config import refetch_frequency
+from parsers.lib.exceptions import ParserException
+
+"""
+This module is used to read the exchange values on a hourly basis for a day from India West to North, East, South. 
+This module also calculates the total electricity consumption by India West.
+The data is fetched from https://www.wrldc.in using APIs which provide data in JSON format
+India West constitutes of Gujrat, Madhya Pradesh, Chattishgarh, Maharashtra, Goa, DD, DNH
+
+These APIs return the data from midnight 12:00am to current time at which the parser is run  
+"""
+
+IN_WE_PROXY = "https://in-proxy-jfnx5klx2a-el.a.run.app"
+IN_TZ = "Asia/Kolkata"
+
+""" 
+Exchange data parser
+source: https://www.wrldc.in/content/166_1_FlowsonInterRegionalLinks.aspx
+sample data:
+[{
+    "Region_Id": 2,
+    "Region_Name": "WR-ER",
+    "Export_Ttc": 25000.0,
+    "Import_Ttc": 25000.0,
+    "Long_Term": 955.0,
+    "Short_Term": -125.0,
+    "Px_Import": 35.0,
+    "Px_Export": 0.0,
+    "Total": 322.0,
+    "Current_Loading": -1052.0,
+    "lastUpdate": "2023-04-05 12:24:09 PM"
```

Comment:
Could you just give a quick explination to what the diffrent fields here are? Are any of them the "realtime" capacity?